content/english/python-guess-number/user-input/_index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Create the game"
+draft: false
+weight: 2
+---
+
+This game will use some of the concepts you learned in the Python Basics section.
+You will need to know how to use the following concepts. If you are not familiar with any of these, please go back to review them.
+- While loops
```

Comment:
Please add links to the relevant basics page, like how's it done with the Python Earsketch curriculum